docs/faq.md

```diff
@@ -21,25 +20,25 @@ import TOCInline from "@theme/TOCInline"
+The [Semgrep Registry](https://semgrep.dev/explore) contains rules from different contributors. Most rules, including all community-contributed rules, are under the [MIT license](https://tldrlegal.com/license/mit-license). Some of the Semgrep Registry rules written by r2c are licensed under the [Semgrep Registry license](../licensing/#semgrep-registry-license).
 
-The source for many r2c-written Registry rules is available at [returntocorp/semgrep-rules](https://github.com/returntocorp/semgrep-rules/). Those licensed under the Semgrep Registry license can’t be resold without r2c’s permission. Since r2c offers a paid, hosted application (Semgrep App), it’s important to have this restriction so other companies, like major cloud providers, can’t just resell our rules as a competing service.
+The source for many r2c-written Registry rules is available at [returntocorp/semgrep-rules](https://github.com/returntocorp/semgrep-rules/). Those rules licensed under the Semgrep Registry license cannot be resold without r2c’s permission. Since r2c offers a paid, hosted application (Semgrep App), it’s important to have this restriction so other companies, like major cloud providers, cannot resell r2c rules as a competing service.
 
 ### Is it ok to run Semgrep or the r2c rules on my work projects?
 
 Yes! Semgrep is safe to run on your private code. The [Semgrep Registry license’s](../licensing/#semgrep-registry-license) commercial restrictions only come into effect if you are _selling_ the rules provided in the semgrep-rules repository. If that’s the case, you’ll need to talk with r2c first to get permission.
 
-### I’m a security professional and want to use the semgrep-rules project with my clients as part of my paid services. Is that ok?
+### I’m a security professional and want to use the Semgrep rules with my clients as part of my paid services. Is that ok?
 
-If you have a typical consulting service and running `semgrep` or `semgrep-rules` is part of your assessments, that’s great and of course feel free to refer your clients to the [Semgrep App](https://semgrep.dev/). But if your entire service is about scanning code and you want to charge for running the `semgrep-rules` repository that r2c and the community created and maintains, you need to discuss this with r2c.
+If you have a typical consulting service and running Semgrep or Semgrep rules is part of your assessments, that’s great and of course feel free to refer your clients to the [Semgrep App](https://semgrep.dev/). But if your entire service is about scanning code and you want to charge for running the `semgrep-rules` repository that r2c and the community created and maintains, you need to discuss this with r2c.
```

Comment:
The text is referring to specific public repositories for `semgrep` and `semgrep-rules`. You can also add a link to those repositories here.

Reply:
Ok! I will change it back and at links to these repositories. @s-santillan

Reply:
I'm not sure about the word "_running_" in this context @s-santillan. Can we say that somebody is "_running_" `semgrep` repository or `semgrep-rules` repository as a part of their assessments? It sounds a bit strange. What do we actually mean by this (in simpler words)? 

Changing to:

```md
If you have a typical consulting service and running [`semgrep` repository](https://github.com/returntocorp/semgrep) or [`semgrep-rules` repository](https://github.com/returntocorp/semgrep-rules)
```